satellite/payments/stripecoinpayments/service.go

```diff
@@ -26,7 +26,7 @@ type Config struct {
 	StripeSecretKey           string        `help:"stripe API secret key" default:""`
 	CoinpaymentsPublicKey     string        `help:"coinpayments API public key" default:""`
 	CoinpaymentsPrivateKey    string        `help:"coinpayments API preivate key key" default:""`
-	TransactionUpdateInterval time.Duration `help:"amount of time we wait before running next transaction update loop" devDefault:"1m" releaseDefault:":30m"`
+	TransactionUpdateInterval time.Duration `help:"amount of time we wait before running next transaction update loop" devDefault:"1m" releaseDefault:"30m"`
```

Comment:
What's the rate limit on coinpayments API?

Reply:
I don't know. Didn't find it in docs, so I will tune this value as we test and develop